packages/dapp-connector/src/storage.ts

```diff
@@ -0,0 +1,37 @@
+import {App} from '@yoroi/types'
+
+export const connectionStorageMaker = (storage: App.Storage): Storage => {
+  const listAllConnections = async () => (await storage.getItem<DappConnection[]>('connections')) || []
+
+  const removeConnection = async (connection: DappConnection) => {
+    const connections = await listAllConnections()
+    const newConnections = connections.filter((c) => !areConnectionsEqual(c, connection))
+    await storage.setItem('connections', newConnections)
+  }
+
+  const addConnection = async (connection: DappConnection) => {
+    const connections = await listAllConnections()
+    const newConnections = [...connections, connection]
+    await storage.setItem('connections', newConnections)
+  }
+
+  return {
+    listAllConnections,
```

Comment:
storage read/save/delete/key, ==> list is manager responsibility

Reply:
on manager {
   connections: {
     list: {
       byName....
       yadada
     }
   }
}